src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -207,6 +214,13 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
             label: "Storage pool",
           }}
         />
+        {!isRemoteStorage && (
+          <ClusterMemberSelector
+            setMember={(member) => {
+              setMember(member);
```

Comment:
this should update the formik state, also initialize it to the location of the volume when the modal opens.